core-bundle/contao/dca/tl_user_group.php

```diff
@@ -405,4 +412,71 @@ public function getExcludedFields()
 
 		return $arrReturn;
 	}
+
+	/**
+	 * Return all editable DCAs
+	 *
+	 * @return array
+	 */
+	public function getEditableDcas()
+	{
+		$processed = array();
+		$files = System::getContainer()->get('contao.resource_finder')->findIn('dca')->depth(0)->files()->name('*.php');
+
+		foreach ($files as $file)
+		{
+			if (in_array($file->getBasename(), $processed))
+			{
+				continue;
+			}
+
+			$processed[] = $file->getBasename();
+
+			$strTable = $file->getBasename('.php');
+
+			System::loadLanguageFile($strTable);
+			$this->loadDataContainer($strTable);
+		}
+
+		$arrReturn = array();
+
+		foreach ($GLOBALS['TL_DCA'] as $k=>$v)
+		{
+			// FIXME: make this configurable
+			if ($k == 'tl_favorites')
+			{
+				continue;
+			}
+
+			// Only handle DC_Table
+			if (!is_a($v['config']['dataContainer'] ?? null, DC_Table::class, true))
```

Comment:
This should handle all tables imho, same as alexf. I also want my Isotope stuff with custom DC in there. Also for tl_files.